data/map planets.txt

```diff
@@ -43,6 +43,13 @@ planet Ada
 		threshold 8000
 		fleet "Large Republic" 16
 
+planet Aetuah
+	attributes "gas giant" "requires: gaslining"
+	landscape land/nasa30
+	description `The close proximity of Aetuah and the neighboring ice moon causes various ebbs and flows in the storms on the gas giant as well as constant reshaping of the moon's surface. The ice moon may have been captured by Aetuah in times past as a rogue planet, and the distance of the gas giant from the system center suggests it might be a rogue planet itself as well. In time, the gravitational force exhibited by Aetuah will tear apart the ice moon, plummeting it into the abyss.`
+	description `	The most striking features of Aetuah are large oval-shaped organisms inhabiting the upper atmosphere. These lifeforms resemble some of the space fauna of nearby systems, and have a strange behavior of slingshotting themselves away from Aetuah once they reach a certain size through some unknown propulsion method.`
+	government Uninhabited
```

Comment:
These should have `security 0` to stop fines from occurring on them.

Reply:
The uninhabited government is hardcoded to skip security checks (unless a non-zero security value is specified by the data).
https://github.com/endless-sky/endless-sky/blob/267f6645b1b368964b86387ad1252cddac4049ad/source/PlayerInfo.cpp#L4123-L4128

Reply:
I pushed the change either way since other Uninhabited planets use it, hopefully that looks good now.

Reply:
I'm pretty sure those other uninhabited planets are using security wrong.

On the "Uninhabited government vs. uninhabited attribute", I think that human space is the only place where uninhabited planets have non-uninhabited governments. The Hai stations, Remnant home system planets and Far'en Lai all use the uninhabited government.

Reply:
So what do I need to change?

Reply:
Just remove the `security 0` line for the planets in this PR.

Reply:
Done.